server/settings/render_settings.py

```diff
@@ -467,6 +471,10 @@ class RenderSettingsModel(BaseSettingsModel):
     renderman_renderer: RendermanSettingsModel = SettingsField(
         default_factory=RendermanSettingsModel,
         title="Renderman Renderer")
+    mayahardware_renderer: MayaHardwareSettingsModel = SettingsField(
+        default_factory=MayaHardwareSettingsModel,
+        title="MayaHardware 2.0 Renderer"
```

Comment:
```suggestion
        title="Maya Hardware 2.0 Renderer"
```
But I even wonder whether the Maya prefix is really needed in the title.